lib/src/rpc.mli

```diff
@@ -9,3 +10,11 @@ val block_put : (string * bool * string, unit) Rpc.t
 val index_get : (string * string, string list option) Rpc.t
 
 val index_put : (string * string * string list, unit) Rpc.t
+
+val metadata_put : (string * string, unit) Rpc.t
+
+(** Transform the response of the block_get pipe RPC into an intelligible stream
+    of the file content and executable boolean *)
+val decode_block_get :
```

Comment:
I think we should make the response type `(string, bool) Either.t` abstract so that we don't need to change the API when we improve the things in future.